File: solutions/2589-maximum-value-of-a-string-in-an-array/solution.py
class Solution(object):
    def maximumValue(self, strs):
        """
        :type strs: List[str]
        :rtype: int
        """
        ls=[]
        for i in strs:
            if not i.isdigit():
                ls.append(len(i))
            else:
                ls.append(int(i))
        return max(ls)
        

File: solutions/2589-maximum-value-of-a-string-in-an-array/solution.java
class Solution {
    public int maximumValue(String[] strs) {
        int n=0;
        for(String s:strs)
        {
            if(s != null && s.matches("\\d+"))
                n=Math.max(n,Integer.parseInt(s));
            else
                n=Math.max(n,s.length());
        }
        return n;
    }
}
